Add cudnn activation ops

xiaomengy opened the pull request:
Summary: Add cudnn activation ops

Differential Revision: D8818013

houseroad commented on caffe2/python/operator_test/activation_ops_test.py:
```diff
@@ -8,35 +8,123 @@
+        self.assertDeviceChecks(dc, op, [X], [0])
+        self.assertGradientChecks(gc, op, [X], 0, [0])
+
+    @unittest.skipIf(not workspace.has_gpu_support,
```
Nice!

houseroad commented on caffe2/python/operator_test/activation_ops_test.py:
```diff
@@ -8,35 +8,123 @@
+    @given(X=hu.tensor(elements=st.floats(-3.0, 3.0)),
+           n=st.floats(min_value=0.5, max_value=2.0),
+           in_place=st.booleans(), **hu.gcs)
+    def test_relu_n(self, X, n, in_place, gc, dc):
```
Shall we also add a case without specifying the n to test the default value 6?

houseroad reviewed (approved):
Thanks for adding CuDNN implementation in activation ops and cleaning the codebase.

BIT-silence commented:
@pytorchbot retest this please

xiaomengy committed:
Add cudnn activation ops (#9379)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/9379

Add cudnn activation ops

Reviewed By: houseroad

Differential Revision: D8818013

fbshipit-source-id: 88aa332b8a17345c798ad65c964bc05b79154ca8
